lib/tool_shed/grids/repository_grids.py

```diff
@@ -1664,16 +1657,14 @@ def get_value( self, trans, grid, repository_metadata ):
                             if tool_id and version:
                                 tool_tups.append( ( tool_id, version ) )
                         sorted_tool_tups = sorted( tool_tups, key=lambda tool_tup: tool_tup[ 0 ] )
-                        num_tool_tups = len( sorted_tool_tups )
-                        for index, tool_tup in enumerate( sorted_tool_tups ):
-                            tool_id = tool_tup[ 0 ]
-                            version = tool_tup[ 1 ]
+                        tool_line = []
```

Comment:
This line should be moved at the start of the function, otherwise the return breaks when some `if` is not satisfied.